WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -6,16 +6,14 @@ def index
     respond_to do |format|
       format.html
       format.js do
-        persons = Person.joins("JOIN Countries ON countryId = Countries.id")
+        persons = Person.order(:name, :countryId)
         if params[:region] != "all"
-          persons = persons.where("countryId = :region OR continentId = :region", region: params[:region])
```

Comment:
For example, our Competition class has this `belongs_to_region` scope that does this (apparently) painful join on countries

@larspetrus, could you create a similar `belongs_to_region` scope for the Person model? For bonus points, you could try to share this scope logic between the `Person` model and the `Competition` model, but I would be fine with us doing that in a later PR.